.github/workflows/linter.yml

```diff
@@ -13,4 +13,4 @@ jobs:
           pip install ruff
 
       - name: Run Ruff Linter
-        run: ruff check --exclude "templates","__init__.py"
+        run: ruff check
```

Comment:
This is actually bad, as we have template files that are not supposed to be checked.